src/app/global/collection-wrapper/CollectionRoutesWrapper.jsx

```diff
@@ -0,0 +1,134 @@
+    }
+
+    getCollectionDetails = collectionID => {
+        return collections.get(collectionID).then(response => {
+            const workingOn = this.mapCollectionResponseToWorkingOnState(response);
+            this.props.dispatch(updateWorkingOn(workingOn.id, workingOn.name, workingOn.url, response, workingOn.error));
+        }).catch(error => {
+            switch(error.status) {
+                case(401): {
+                    // do nothing - this is handled by the request function itself
+                    break;
+                }
+                case(404): {
+                    const notification = {
+                        type: 'neutral',
+                        message: "Collection couldn't be found so you've been redirected to the collections screen",
+                        autoDismiss: 5000
+                    };
+                    notifications.add(notification);
+                    this.props.dispatch(replace(`${this.props.rootPath}/collections`));
+                    break;
+                }
+                case(403): {
+                    const notification = {
+                        type: 'neutral',
+                        message: "You don't have permissions to access this collection so you've been redirect to the collections screen",
+                        autoDismiss: 5000
+                    };
+                    notifications.add(notification);
+                    this.props.dispatch(replace(`${this.props.rootPath}/collections`));
+                    break;
+                }
+                case('FETCH_ERR'): {
+                    const notification = {
+                        type: 'warning',
+                        message: "There was a network error whilst getting details about this collection, please check your connection and refresh the page",
+                        autoDismiss: 5000
+                    };
+                    notifications.add(notification);
+                    break;
+                }
+                default: {
+                    const notification = {
+                        type: 'warning',
+                        message: "An unexpected error occurred whilst getting details about this collection, please refresh the page",
+                        autoDismiss: 5000
+                    };
+                    notifications.add(notification);
+                    break;
+                }
+            }
+        })
+    }
+
+    mapCollectionResponseToWorkingOnState = collection => {
+        log.event("collection response to 'working on' state");
+        try {
+            return {
+                id: collection.id,
+                name: collection.name,
+                url: `${this.props.rootPath}/collections/${collection.id}`,
+                error: false
+            }
+        } catch (error) {
+            log.event("error mapping collection response to 'working on' state", log.error(error), 
```

Comment:
Might be worth using the same casing of 'error/Error' in copying the same error message for `log.event` and `console.error`

Reply:
Fixed